py_ecc/bls/point_compression.py

```diff
@@ -121,16 +122,19 @@ def decompress_G1(z: G1Compressed) -> G1Uncompressed:
 #
 def modular_squareroot_in_FQ2(value: FQ2) -> Optional[FQ2]:
     """
-    ``modular_squareroot_in_FQ2(x)`` returns the value ``y`` such that ``y**2 % q == x``,
+    Given value=``x``, returns the value ``y`` such that ``y**2 % q == x``,
     and None if this is not possible. In cases where there are two solutions,
     the value with higher imaginary component is favored;
     if both solutions have equal imaginary component the value with higher real
     component is favored.
     """
     candidate_squareroot = value ** ((FQ2_ORDER + 8) // 16)
-    check = candidate_squareroot ** 2 / value
+    check = candidate_squareroot**2 / value
     if check in EIGTH_ROOTS_OF_UNITY[::2]:
```

Comment:
This can be put in a separate PR/issue, but I believe this should be: `EIGHTH_ROOTS_OF_UNITY`

Reply:
fine in this PR I think, updated everywhere